### PR TITLE
feat: adding pt-br translation to README

### PR DESCRIPTION
**Pull request by Azzyew**
I created a README.pt-br.md and linked it on the original README.md

I really believe we should share the knowledge the best way we can.

### CODE REVIEW

**Comment by StorytellerCZ** [comment]
If we go with translating, then we should probably adopt some framework for this so it is easy to get other languages in without having crazy amount of files in the repo.

**Comment by filipenevola** [comment]
Hi @Azzyew, thank you for your translation.

I'm Brazilian so I'm biased here to make a decision about this specific translation hehe.

But what people are doing to have READMEs in many languages? I'm really not aware if we have a solution for this.

Maybe we could have a folder called `translations` and then inside this folder we can replicate the same structure for each language and ask the community to fill those in. Then in the main Readme we could just add Country flags for these folders.

What do you think?

For example:
```
- translations/pt-br/
  - README.md
  - CONTRIBUTING.md
  - DEVELOPMENT.MD
  - ...
- translations/en-us/
  - README.md
  - ...
```

**Comment by Azzyew** [comment]
@filipenevola that's a great idea! I'm gonna make those changes and commit when they're ready

**Comment by namenotrequired** [comment]
Great initiative.

We need to be careful. Translations tend to get out of date.

We have enough contributors who speak pt-BR (like me). We just may need some minimal structure to make it more likely that future updates to the original files will be translated.

**Comment by Azzyew** [comment]
translations folder created!

File: translations/PT-BR/README.pt-br.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+Interessado em ajudar ou contribuir para o Meteor? Esses recursos vão ajudar:
+
+* [Guia principal de desenvolvimento](DEVELOPMENT.md)
+* [Guias de contribuição](CONTRIBUTING.md)
+* [Requisição de recursos](https://github.com/meteor/meteor-feature-requests/)
+* [Rastreador de Issues](https://github.com/meteor/meteor/issues)
+
+## Desinstalando Meteor
+
+Sem ser pelo pequeno script launcher shell, o Meteor se instala dentro de seu diretório principal. Para desinstalar Meteor, use:
```
Comment (filipenevola) [inline translations/PT-BR/README.pt-br.md:66]: It seems like a literal translation `pequeno script launcher`

File: translations/PT-BR/README.pt-br.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+[![TravisCI Status](https://travis-ci.org/meteor/meteor.svg?branch=devel)](https://travis-ci.org/meteor/meteor)
+[![CircleCI Status](https://circleci.com/gh/meteor/meteor/tree/devel.svg?style=shield&circle-token=c2d3c041506bd493ef3795ffa4448684cfce97b8)](https://circleci.com/gh/meteor/meteor/tree/devel)
+
+Meteor é um ambiente ultra-simples para construir aplicações web modernas.
+
+Com Meteor você escreve aplicações:
+
+* em JavaScript moderno
+* que manda dados pelo wire, ao invés de HTML
```
Comment (filipenevola) [inline translations/PT-BR/README.pt-br.md:11]: Wire was not translated here

File: translations/PT-BR/README.pt-br.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+* usando sua escolha de bibliotecas open-source populares
+
+Tente um tutorial para começar:
+ * [React](https://react-tutorial.meteor.com) - novo
+ * [Blaze](https://www.meteor.com/tutorials/blaze/creating-an-app)
+ * [Angular](https://www.meteor.com/tutorials/angular/creating-an-app)
+ * [Vue](https://www.meteor.com/tutorials/vue/creating-an-app)
+ * [Svelte](https://www.meteor.com/tutorials/svelte/creating-an-app)
+
+Depois, leia a [guia](https://guide.meteor.com) e a [documentação](https://docs.meteor.com/).
```
Comment (filipenevola) [inline translations/PT-BR/README.pt-br.md:21]: It should be "o" guia

**Review by filipenevola — CHANGES_REQUESTED** [review]
I like the idea but the translation has some issues. Please check my comments.

**Comment by filipenevola** [comment]
I'm really afraid that these translations would be not updated.

**Comment by filipenevola** [comment]
Hi, as we are not having many discussions around this PR I'm closing it.

Of course, if we have new ideas we can re-open it. No problem.

**Pull request closed without merging**